var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -124,6 +124,8 @@ class Geant4(CMakePackage):
     depends_on("motif", when="+motif")
     depends_on("qt@5: +opengl", when="+qt")
 
+    # Fixes bug 2444 on 10.7.2: https://bugzilla-geant4.kek.jp/show_bug.cgi?id=2444
```

Comment:
Since this is a known bug in the software, we generally prefer the patches be upstreamed and can be pulled using a URL.

Reply:
Any chance of getting that done?

Reply:
I cannot say for sure but I have reasons to believe that is not easy.
1) They have already patched it in 10.7.3 and recommend the upgrade (which changes a lot of other things that we don't necessarily want)
2) I've not seen this sort of thing from Grant4 team before. They tend to release only major version patches not individual bug fixes
3) I'm not even sure where they actively develop. They only import major versions to git from some private repo it seems.

Reply:
Okay.  Thanks for the response.